packages/components/organisms/f-checkout/src/components/Header.vue

```diff
@@ -0,0 +1,182 @@
+<template>
+    <div
+        data-test-id='header-component'
```

Comment:
I'd update these test id's too, so they definitely never clash with the main site header

Reply:
And nit: we should use `"` instead of `'` here.